README.md

```diff
@@ -0,0 +1,52 @@
+# Difflint #
+
+Do you have a codebase with inconsistent code style, and you want to
+start checking your commits with a linter; but you don't want to rewrite
+the entire repository, since it will mess up your history and the
+ability to `git bisect`?
+
+**Difflint** will check only the changes that you make in each commit.
+It's suitable for using as a Git pre-commit hook.
+
+## Linters supported ##
+
+- [JSCS](http://jscs.info) (JavaScript)
+- [JSHint](http://jshint.com/docs) (JavaScript)
+
+## Requirements ##
+
+- Python 3.3 or later
+- JSCS
+- JSHint
+- NPM for installing JSCS and JSHint
+
+## Installation ##
+
+- Clone this repository, and run `python3 setup.py install`.
```

Comment:
Sounds good on all counts. Will have to mull over this more to see if there is any way we can make the installation process easier.